drivers/src/nrfx_nfct.c

```diff
@@ -528,6 +528,36 @@ nrfx_err_t nrfx_nfct_tx(nrfx_nfct_data_desc_t const * p_tx_data,
     return NRFX_SUCCESS;
 }
 
+nrfx_err_t nrfx_nfct_tx_bits(nrfx_nfct_data_desc_t const * p_tx_data,
+                             nrf_nfct_frame_delay_mode_t   delay_mode)
+{
+    NRFX_ASSERT(p_tx_data);
+    NRFX_ASSERT(p_tx_data->p_data);
+
+    if (p_tx_data->data_size == 0)
+    {
+        return NRFX_ERROR_INVALID_LENGTH;
+    }
+
+	// calculate the buffer length in bytes based on the number of bits,
```

Comment:
Indentation of this block is unaligned with the rest of the function body (should be 4).